azure-quantum/azure/quantum/qiskit/backends/backend.py

```diff
@@ -75,6 +75,15 @@ def _translate_input(self, circuit, data_format, input_params):
         if not "arguments" in input_params:
             input_params["arguments"] = []
 
+        if input_params.get("auto_transpile", False):
+            from qiskit import transpile
```

Comment:
This should be imported at the top with the rest of the qiskit imports

Reply:
Sounds good.